package.json

```diff
@@ -47,6 +47,7 @@
     "@types/uuid": "^8.3.0",
     "@types/yup": "^0.29.11",
     "ajv-keywords": "^4.0.0",
+    "apollo-absinthe-upload-link": "^1.7.0",
     "apollo-link-token-refresh": "^0.3.2",
     "apollo-upload-client": "^14.1.3",
```

Comment:
@mdshamoon We should remove this if not used.

Reply:
Removed the package